server/app/assets/javascripts/file_upload.ts

```diff
@@ -26,12 +28,25 @@ function validateFileUploadQuestions(formEl: Element): boolean {
     )
     const isValid = fileInput.value != ''
 
-    // Toggle the error div if invalid.
     const errorDiv = question.querySelector('.cf-fileupload-error')
     if (errorDiv) {
-      // TODO(#1878): Update button aria attributes on error.
+      // Toggle the error div if invalid.
       errorDiv.classList.toggle('hidden', isValid)
     }
+    if (errorDiv && !isValid && !wasSetInvalid) {
+      // Add extra aria attributes to input if there is an error.
+      const errorId = errorDiv.getAttribute('id')
+      if (errorId) {
+        // Only allow this to be done once so we don't repeatedly append the error id.
+        wasSetInvalid = true
+        fileInput.setAttribute('aria-invalid', 'true')
+        const ariaDescribedBy = fileInput.getAttribute('aria-describedBy')
+        fileInput.setAttribute(
+          'aria-describedBy',
```

Comment:
question (blocking): does the b in "by" need to be lower case?

Reply:
No, either is fine. Looking at the docs, it automatically all gets lowercased: 
https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute#lower_casing
https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute#parameters

And I see that the HTML that's rendered by j2html actually is all lowercase too. I'll go through and just change everything to `aria-describedby` since it's unnecessary that I capitalized the `B`. Will send that out separately.